.github/workflows/development.yml

```diff
@@ -1,4 +1,4 @@
-name: Build and Test
+name: Development
```

Comment:
This is so where workflows are run map 1:1 with the stages we discussed. Will need to change out `Test` environment to `Development` as well

Reply:
(Already changed it)